### PR TITLE
3780 migrating metadataprops e2e to puppeteer

### LINKED ISSUE #3780: Migrating metadataprops e2e test
The task is to migrate `nightmare/suite1/metadataprops.spec.js` to the puppeteer suite.

### PR DESCRIPTION
**Pull request by LaszloKecskes**
fixes #3780 

PR checklist:
- [ ] Update READ.me ?
- [ ] Update API documentation ?

QA checklist:
- [ ] Smoke test the functionality described in the issue
- [ ] Test for side effects
- [ ] UI responsiveness
- [ ] Cross browser testing
- [ ] Code review

### CODE REVIEW

File: e2e/suites/metadataprops.test.ts [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+describe('Metadata Properties', () => {
+  beforeAll(async () => {
+    await insertFixtures();
+    await proxyMock();
+    await disableTransitions();
+  });
+
+  it('should log in as admin then click the settings nav button.', async () => {
+    await adminLogin();
+    await waitForNavigation(expect(page).toClick('a', { text: 'Account settings' }));
```
Comment (Zasa-san) [inline e2e/suites/metadataprops.test.ts:19]: I the waitForNavigation necessary? Otherwise you can do a expect directly.

Comment (Zasa-san) [inline e2e/suites/metadataprops.test.ts:19]: Just curious, why is this needed here?

File: e2e/suites/metadataprops.test.ts [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+      '.form-group.multidaterange .multidate-item:nth-of-type(2) div.DatePicker__To input',
+      '12/09/1964'
+    );
+    await expect(page).toFill('.form-group.markdown textarea', '***smile***');
+    await expect(page).toFill('.form-group.link #label', 'Huridocs');
+    await expect(page).toFill('.form-group.link #url', 'https://www.huridocs.org/');
+
+    await expect(page).toClick('button', { text: 'Save' });
+    await expect(page).toClick('div.alert-success');
+  }, 60000);
```
Comment (Zasa-san) [inline e2e/suites/metadataprops.test.ts:98]: Locally it works without the timeouts, is it failing on GH?

File: e2e/suites/metadataprops.test.ts [hunk 3]
```diff
@@ -0,0 +1,166 @@
+/* eslint-disable max-statements */
```
Comment (Zasa-san) [inline e2e/suites/metadataprops.test.ts:1]: Nit: I think this could be avoided by wrapping the longer `its` into a `describe`, with `its` that cover some properties, like: "it should fill all the dates", "it should fill all the selects", and then "it should have all the texts filled correctly", etc.

File: e2e/suites/metadataprops.test.ts [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+    expect(propertyList.length).toBe(17);
+  });
+
+  it('should create a template with all the properties.', async () => {
+    await expect(page).toClick('a', { text: 'Templates' });
+    await expect(page).toClick('a', { text: 'Add template' });
+    await expect(page).toFill('input[name="template.data.name"]', 'All props');
+
+    const propertyAddButtons = await page.$$('.property-options-list li button');
+    //intentionaly leaving geolocation and the fields after out of the test.
```
Comment (Zasa-san) [inline e2e/suites/metadataprops.test.ts:36]: Could we expand this comment to explain that this is to avoid the "Violated Articles" property?

**Review by Zasa-san — CHANGES_REQUESTED** [review]
LGTM! Only left some minor comments.

File: e2e/suites/metadataprops.test.ts (hunk 1 above)
Comment (LaszloKecskes) [reply e2e/suites/metadataprops.test.ts:19]: In the prod build (and consequently, on the CI) it picks up the link to early, and the next check fails. See this run:
https://github.com/huridocs/uwazi/runs/2960379218#step:15:44

File: e2e/suites/metadataprops.test.ts (hunk 2 above)
Comment (LaszloKecskes) [reply e2e/suites/metadataprops.test.ts:98]: This is something that was used in the previous implementation. I'm not sure about the reason, since I see now actions there that might hang. Probably you are right and we can remove this.

File: e2e/suites/metadataprops.test.ts (hunk 3 above)
Comment (LaszloKecskes) [reply e2e/suites/metadataprops.test.ts:1]: In general, I have asked Konz about the eslint, and he said that in tests it's accepted to go over the line limit. 

Specifically for this case, I think it's nice to have them together, in the original order.

File: e2e/suites/metadataprops.test.ts (hunk 4 above)
Comment (LaszloKecskes) [reply e2e/suites/metadataprops.test.ts:36]: Sure.

**Review by Zasa-san — APPROVED** [review]
LGTM!
